.profile.d/001_fixup_apt.sh

```
@@ -0,0 +1,8 @@
+# .profile.d/001_fixup_apt.sh
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)